auraed/src/cells/cell_service/cell_service.rs

```diff
@@ -214,11 +234,25 @@ impl CellService {
         info!("CellService: stop() executable_name={:?}", executable_name,);
 
         let mut executables = self.executables.lock().await;
-        let _exit_status = executables
+        let executable = executables
             .stop(&executable_name)
             .await
             .map_err(CellsServiceError::ExecutablesError)?;
 
+        let pid = executable
+            .pid()
+            .map_err(CellsServiceError::Io)?
+            .expect("pid")
```

Comment:
This will always panic since an `Executable` in the `Stopped` `ExecutableState` will always return `None`. You can get the `pid` prior to stopping it.

Reply:
ack